### ISSUE TITLE
Loss can not go down.

### ISSUE THREAD

**Issue by Li-brua** [LukasStruppek/Plug-and-Play-Attacks#2]
Hello, I executed this command: python attack.py -c=configs/attacking/default_attacking.yaml, and load Stylegan2 weight pretrained on FFHQ1024, to attack Resnet18 train on CelebA256，but I found that under Adam's optimization, loss could not go down at all.

<img width="842" alt="571d491de1c351c94124b4a667eab52" src="https://user-images.githubusercontent.com/112678612/226827474-14f5c888-9291-4a83-8e7e-9816f0bd5259.png">
<img width="657" alt="d766254e49ad2713e5d5b2f39b73f67" src="https://user-images.githubusercontent.com/112678612/226827514-b94abeae-023c-4f5f-be0a-9bd1a5aaaba5.png">

I have no idea about about this problem......In attacking config，I just changed batchsize and weight path.

**Comment by LukasStruppek** [comment]
Hey,

that seems strange. Could you maybe try to re-clone the project and run it again? I have currently no access to my machine, so I can't try it right now. But a couple of weeks ago, it worked quite fine for me. But I also used the WandB integration to load the models.

I the problem still consists in a fresh repo, I will try it next week in my machine.

**Comment by Li-brua** [comment]
thank you for you answer, I have already solve this problem. The main problem is the training code of the target model, when the trained model saves the weight, it saves some unwanted data, resulting in an error when attack.py loading the weight, I added the strict=False parameter, only such a problem occurs, and then I modified the weight saving code of your training model, only save the model weight information, strict=True can also load the target model weight normally, and the problem is solved.

**Comment by LukasStruppek** [comment]
Ah, I see. The additional data is saved to allow continuing the training procedure. Thanks for your feedback, I might improve that part of the code.
